Source/ASTableView.mm

```diff
@@ -1581,6 +1581,11 @@ - (void)rangeController:(ASRangeController *)rangeController didUpdateWithChange
 
 #pragma mark - ASDataControllerDelegate
 
+- (BOOL)dataControllerShouldAutomaticallyLayoutNodes:(ASDataController *)dataController
```

Comment:
Does this have a use case yet where NO is used?  If not, should we remove it and add it once a use case exists?